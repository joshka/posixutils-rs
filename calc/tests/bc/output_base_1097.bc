obase = 1097
0
1
10783
103072.2109348
quit
