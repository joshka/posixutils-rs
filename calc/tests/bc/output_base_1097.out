0
 0001
 0009 0910
 0093 1051.0231 0433 0917
